modules/fts/pages/fts-query-string-syntax-field-scoping.adoc

```diff
@@ -14,9 +15,11 @@ You can adjust this by prefixing items with `+` or `-`.
 * Prefixing with `+` places that item in the MUST portion of the boolean query.
 * Prefixing with `-` places that item in the MUST NOT portion of the boolean query.
 
+Note, the above syntax is a simple and easy way to create xref:fts-supported-queries-compound-query.adoc[Compound Queries] in Search.
+
 === Example
 
 For example, `description:pool` performs a xref:fts-supported-queries-match.adoc[match query] for the term `pool`, in the `description` field.
 
 For example, `+description:pool -continental breakfast` performs a boolean query that MUST satisfy the match query for the term `pool` in the `description` field, MUST NOT satisfy the match query for the term `continental` in the `default` field, and SHOULD satisfy the match query for the term `breakfast` in the `default` field.
-Result documents satisfying the SHOULD clause score higher than those that do not.
\ No newline at end of file
+Result documents satisfying the SHOULD clause score higher than those that do not.
```

Comment:
So I want to make sure I understand this correctly: a document that matches:

- Pool in the description field
- No continental in the default field

will be scored as less than another document that also has breakfast in the default field?

Reply:
No I believe that the scoring goes up more if we hit a +<want_text> verses if we are missing a -<do_not_want_text>

Reply:
I'm just trying to figure out how the SHOULD fits in, and why we specifically called out the SHOULD clause?

Reply:
SHOULD just means +<want_text> has higher impact than -<do_not_want_text>
Consider the query
**+chaise -pool**
Now if we have a document that has one (1) occurance of "chaise" and one (1) occurance of "pool"
The **+chaise** will have higher impact.
Now if we have a document that has one (1) occurance of "chaise" and five (5) occurrences of "pool"
The **-pool** will have higher impact.